spec.bs

```diff
@@ -285,6 +285,16 @@ Each {{URLPattern}} object has an associated <dfn for=URLPattern>hash component<
   1. Return the result of [=match=] given [=this=], |input|, and |baseURL| if given.
 </div>
 
+<h3 id=urlpattern-internals>Internals</h3>
+
+A {{URLPattern}} is associated with multiple <dfn>component</dfn> [=structs=].
```

Comment:
Probably the "Each URLPattern object has an associated protocol component, a component, initially null" and friends should move into this section as well?

Reply:
I guess I thought that those should stick with `{{URLPattern}}` definition as its kind of like internal properties.

Reply:
Hmm yeah, I see the division you were trying to make now. Seems fine.